index.html

```diff
@@ -10,7 +10,7 @@
   <body>
     <section class="hero">
       <nav>
-        <img class="logo" src="assets/img/Logo.png" />
+       <a href="#Home"><img class="logo" src="assets/img/nav-logo.svg" /></a> 
         <ul>
           <li><a href="#home" class="active">Home</a></li>
           <li><a href="#explore">Explore</a></li>
```

Comment:
Make the links click and scroll to the relevant sections. e.g #explore should take the user to the About section, "#contact" should take the usetr to the footers ection etc.